src/components/pill-selector-dropdown/PillSelectorDropdown.js

```diff
@@ -121,6 +124,9 @@ class PillSelectorDropdown extends React.Component<Props, State> {
         // "Select" the pills
         if (pills.length > 0) {
             onSelect(pills);
+            if (onPillCreate) {
+                onPillCreate(pills);
```

Comment:
Cant use onSelect?

Reply:
No. OnPillCreate is basically for paste,enter,blur, these 3 have different behavior as onSelect. So what it does is, after I do paste,ener,blur, the pills created is just the raw email addresses, and then it calls to fetch the user info and then replace the text inside the pill with the user name with the domain if it's an external collab

Reply:
But onSelect is actually selecting the dropdown which already has the user name option

Reply:
Yeah, I recommended this pattern since we don't want to conflate the `onSelect` and `onInput` events with this post-creation pill resolving event.

Reply:
Are we guaranteed that pills will be created? I think onSelect() be async if validation needs to run on the server?